### ISSUE TITLE
Error: (94, 0) Declaring custom 'clean' task when using the standard Gradle lifecycle plugins is not allowed? Cannot resolve symbol MavenDeploy?

### ISSUE THREAD

**Issue by scheung38** [braintree/braintree_android#153]
### General information

* SDK/Library version: 
* Environment:     Sandbox 
* Android Version and Device: 

### Issue description

/braintree_android/build.gradle
Error: (94, 0) Declaring custom 'clean' task when using the standard Gradle lifecycle plugins is not allowed?

Cannot resolve symbol 'MavenDeployment'?

**Comment by quinnjn** [comment]
Thanks for the report. We'll take a look into this.

**Comment by quinnjn** [comment]
I do see a warning when using the gradle wrapper that the custom clean task is deprecated in Gradle 3.0. We'll work on fixing this.

```
$ ./gradlew tasks
Starting a new Gradle Daemon for this build (subsequent builds will be faster).
Defining custom 'clean' task when using the standard Gradle lifecycle plugins has been deprecated and is scheduled to be removed in Gradle 3.0
```

I haven't ran into the other issue you mentioned. Can you provide us steps to reproduce the `Cannot resolve symbol 'MavenDeployment'?` error?

**Comment by febinmathew** [comment]
Check out [this link](http://techusall.com/solved-android-studio-error24-0-declaring-custom-clean-task-using-standard-gradle-lifecycle-plugins-not-allowed/). Everything is explained in it. Sure this will help !

**Comment by quinnjn** [comment]
@scheung38 We just published [2.5.4](https://github.com/braintree/braintree_android/releases/tag/2.5.4) which should take care of the clean task error.

Feel free to re-open this ticket with reproducing steps for your other issue: `Cannot resolve symbol 'MavenDeployment'`
